media/src/activity/activity-map-pane-panels.tsx

```diff
@@ -386,13 +396,11 @@ const FacultySubPanel: React.FC<FacultySubPanelProps> = ({
             </section>
 
             <section className={'lt-pane-section'}>
-                <h3>Reflection</h3>
+                <h3>Reflection what is this?</h3>
```

Comment:
@zmustapha Did you mean to leave this in?

Reply:
Oops, I missed that one. Will remove